Test/Unit/Visitor/Arithmetic.php

```diff
@@ -65,7 +65,7 @@ public function case_visitor_exhaustively()
                     new Regex\Visitor\Isotropic(
                         new LUT\Sampler\Random()
                     ),
-                    9
+                    7
```

Comment:
I changed this value because with `9` the test suite generated more than a million expressions and I could not reach the end of the test... way tooooooo long :)